drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/phases/TornadoFPGAPragmas.java

```diff
@@ -53,7 +53,19 @@
 
 public class TornadoFPGAPragmas extends BasePhase<TornadoHighTierContext> {
 
+    /*
+     * The default factor number for loop unrolling is 4, because this number
+     * yielded higher performance on Intel FPGAs.
+     */
```

Comment:
Is there any paper as reference? I think Michalis proved this in the Programming 2021 Journal

Reply:
Good point. @mikepapadim if you can forward me the link that mentions it and I can add it.